lib/llvmopencl/Workgroup.cc

```diff
@@ -1298,7 +1315,7 @@ LLVMValueRef Workgroup::createArgBufferLoad(LLVMBuilderRef Builder,
     LLVMTypeRef DestTy = LLVMPointerType(ParamType, DeviceArgsASid);
     LLVMValueRef ArgOffsetBitcast =
         LLVMBuildPointerCast(Builder, ArgByteOffset, DestTy, "arg_ptr");
-#ifdef LLVM_OLDER_THAN_14_0
+#ifndef LLVM_OPAQUE_POINTERS
```

Comment:
This breaks if LLVM_OPAQUE_POINTERS undefined and LLVM 14? Produces opaque pointers?